packages/ui/src/components/molecules/PremiumFeature/PremiumFeature.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react';
+import { Crown } from 'lucide-react';
+
+import { Card, CardContent, CardFooter, CardHeader, Image } from '@/components';
+
+export const PremiumFeature = ({ footer, content }: { footer: ReactNode; content: ReactNode }) => (
```

Comment:
No need for a change now. Pretty sure this is how children are typed.
```suggestion
export const PremiumFeature = ({ footer, content }: { footer: ReactNode | ReactNode[]; content: ReactNode  | ReactNode[]; }) => (
```

Reply:
As we saw, this is not necessarily the correct way to type.